### PR TITLE
feat: allow dotnet prerelease

### PR DESCRIPTION
**Pull request by acesyde**
Allow prerelease

### CODE REVIEW

File: src/backend/dotnet.rs [hunk 1]
```diff
@@ -28,9 +28,10 @@ impl Backend for DotnetBackend {
         let feed_url = self.get_search_url()?;
 
         let feed: NugetFeedSearch = HTTP_FETCH.json(format!(
-            "{}?q={}&packageType=dotnettool&take=1",
+            "{}?q={}&packageType=dotnettool&take=1&prerelease={}",
```
Comment (jdx) [inline src/backend/dotnet.rs:31]: are there other flags we might be able to send? I'm thinking of a setting that might allow more functionality like `MISE_DOTNET_PACKAGE_FLAGS=prerelease` or something

Comment (acesyde) [reply src/backend/dotnet.rs:31]: There is no other parameters to send to the nuget api, I let you decide what you prefer and I can implement it 👍

Comment (acesyde) [reply src/backend/dotnet.rs:31]: I will check something and if it's better to implement your solution for the future , do not merge it yet please

Comment (jdx) [reply src/backend/dotnet.rs:31]: Looking at this again, should this actually be a tool option? Or maybe we support both?

https://mise.jdx.dev/dev-tools/#tool-options

Comment (acesyde) [reply src/backend/dotnet.rs:31]: Hmm good idea, I think I'll implement it, it's better if it works the same everywhere

Comment (acesyde) [reply src/backend/dotnet.rs:31]: After investigation the `prerelease` is at the `_list_remote_versions` level not at the `install_version_` so doesn't have access to `tv.request.options()`

**Pull request merged**